autoendpoint/src/extractors/subscription.rs

```diff
@@ -389,27 +389,6 @@ fn validate_vapid_jwt(
         return Err(VapidError::FutureExpirationToken.into());
     }
 
-    let aud = match Url::from_str(&token_data.claims.aud) {
-        Ok(v) => v,
-        Err(_) => {
-            error!("Bad Aud: Invalid audience {:?}", &token_data.claims.aud);
-            metrics.clone().incr("notification.auth.bad_vapid.aud");
-            return Err(VapidError::InvalidAudience.into());
-        }
-    };
-
-    let domain = &settings.endpoint_url();
-
-    if domain != &aud {
```

Comment:
I gather jsonwebtoken will now do an equivalent check against our `settings.vapid_aud` (via `validation.set_audience()`) -- however I don't see us assigning this new `AUTOEND__VAPID_AUD` setting [anywhere](https://github.com/mozilla-it/webservices-infra/blob/main/autopush/k8s/autoendpoint/templates/configmap.yaml), which I think means it's [defaulting to incorrect values](https://github.com/mozilla-services/autopush-rs/blob/master/autoendpoint/src/settings.rs#L73) (not `settings.endpoint_url()`). 

Do we even need `settings.vapid_aud` -- might we always set something along the lines of `validation.set_audience(&[settings.endpoint_url()])` instead?

Reply:
and prod's basically `endpoint_url()` for reference [here](https://github.com/mozilla-it/webservices-infra/blob/fb647cb8d278b0ea8dde65d56a9452986678ab1e/autopush/k8s/autoendpoint/values-prod.yaml#L26)